### PR TITLE
Adds image_url field to Pokemon model

### PR DESCRIPTION
**Pull request by jaenia**
[As a Trainer I see the details of a settled battle](https://trello.com/c/SpK1rW4K/9-as-a-trainer-i-see-the-details-of-a-settled-battle)

### Description
This PR adds `image_url` field to `Pokemon` model.

### CODE REVIEW

**Review by filipeximenes — COMMENTED** [review]
Any reason you are not using `ImageField`?
https://docs.djangoproject.com/en/3.1/ref/models/fields/#django.db.models.ImageField

**Comment by jaenia** [comment]
> Any reason you are not using `ImageField`?
> https://docs.djangoproject.com/en/3.1/ref/models/fields/#django.db.models.ImageField

No. Actually, I had doubts to choose the best approach. With the ImageField, I'll save the image file in my database, right? I thought that for this case I could put the image url where I need without having it in my database. I'm just considering that this is more simple. Am I wrong? Do you think it can bring any risk? Do you think it's better to have the images in the database?

**Comment by filipeximenes** [comment]
From the link I sent above: 
> ImageField instances are created in your database as varchar columns with a default max length of 100 characters. 

`ImageField` is a `char` field just like `FileField` is. Django does not offers a "blob" field by default.

**Comment by jaenia** [comment]
> From the link I sent above:
> 
> > ImageField instances are created in your database as varchar columns with a default max length of 100 characters.
> 
> `ImageField` is a `char` field just like `FileField` is. Django does not offers a "blob" field by default.

Got it! Will try that then. Thank you.

File: backend/pokemons/models.py [hunk 1]
```diff
@@ -7,6 +7,7 @@ class Pokemon(models.Model):
     attack = models.IntegerField()
     defense = models.IntegerField()
     hit_points = models.IntegerField()
+    image = models.ImageField(blank=True, max_length=100)
```
Comment (filipeximenes) [inline backend/pokemons/models.py]: No need to set the `max_length` here, it's 100 by default

Comment (filipeximenes) [reply backend/pokemons/models.py]: You should set the `upload_to` param. Go read about it in `FileField` docs: https://docs.djangoproject.com/en/3.1/ref/models/fields/#django.db.models.FileField

**Review by filipeximenes — CHANGES_REQUESTED** [review]

File: backend/pokemons/models.py (hunk 1 above)
Comment (jaenia) [reply backend/pokemons/models.py]: Thanks for the references!

**Review by filipeximenes — APPROVED** [review]

**Pull request merged**